File: Data Structure/String/Easy/LET_no1903_Largest Odd Number in String.py
class Solution:
    def largestOddNumber(self, num: str) -> str:
        i = len(num) - 1
        while int(num[i]) % 2 == 0 and i >= 0:
            i -= 1
        
        return num[:i + 1] if i >= 0 else ""

File: Data Structure/String/Easy/LET_no1903_Largest Odd Number in String.cpp
class Solution {
public:
    string largestOddNumber(string num) {
        int i = num.size() - 1;
        while (i >= 0 and int(num[i]) % 2 == 0)
            i--;
        
        return i >= 0 ? num.substr(0, i + 1) : "";
    }
};